book/views.py

```diff
@@ -79,3 +83,17 @@ def edit_book(request, pk):
     else:
         form = BookForm(instance=book)
     return render(request, 'book/edit_book.html', {'form': form, 'pk': pk})
+
+
+class BookAPIView(viewsets.ModelViewSet):
+
+    queryset = Book.objects.all()
+
+    def get_serializer_class(self):
```

Comment:
Just add serializer_class and make one serializer

Reply:
Fixed